playbook.yml

```diff
@@ -3,7 +3,10 @@
   tasks:
   - include_role:
       name: "uperf-bench"
-    when: uperf is defined and uperf.pair > 0
+    when: uperf.pair is defined and uperf.pair > 0
```

Comment:
We'll still need when: uperf is defined, as the user can tinker with the cr and remove the uperf altogether despite instructions.